index/test_petitions_model.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+import calc
+
+class TestPetitionsModel(unittest.TestCase):
+
+	def test_

```

Comment:
If you're able to add unit tests now for the functions that you added in `Petitions` and `User`, that would be great!